README.md

```diff
@@ -19,13 +19,14 @@ This SDK for [Node.js](https://nodejs.org/) was created based on the [Zenvia](ht
 - [Prerequisites](#prerequisites)
 - [Installation](#installation)
 - [Basic Usage](#basic-usage)
-  - [Sending your first message](#sending-your-first-message)
+  - [Send your first message](#send-your-first-message)
+  - [Send a batch](#send-a-batch)
   - [Subscribe for messages](#subscribe-for-messages)
   - [Subscribe for message status](#subscribe-for-message-status)
-  - [Receiving message and message status events](#receiving-message-and-message-status-events)
-  - [Getting messages report](#getting-messages-report)
-  - [Getting flow report](#getting-flow-report)
-  - [Listing your templates](#listing-your-templates)
+  - [Receive message events and message status events](#receive-message-events-and-message-status-events)
+  - [Get a messages report](#get-a-messages-report)
+  - [Get a flow report](#get-a-flow-report)
+  - [List your templates](#list-your-templates)
```

Comment:
Tenho a sensação que gerundio soa melhor aqui nesses verbos, você diria que estava errado do jeito que estava?

Reply:
Eu geralmente não uso gerúndio. Exceto em casos em que a ação está ocorrendo durante um estado tempo informado (ex: Não me mande mensagem. Me ligue, pois estarei estudando). O gerúndio da a sensação de uma ideia sem estado, condição ou instância real. Em outras palavras, que não tem término. É mais uma idéia do que uma ação.
Não considero errado na forma que está apresentada no Readme, mas a conjugação para qual eu mudei é mais _call-to-action_ e implica que uma ação vai ocorrer e terminar naquele contexto.

Reply:
Entendi. Mas acho que é justamente por isso que tenho a sensação que fica melhor no gerúndio, pois é algo sem fim por ocorrer recorrentemente, acho que por isso que tenho a sensação que "Getting flow reports" soa melhor do que "Get a flow report". E a impressão que eu tenho é quue é o mais usual pra documentações mesmo o gerúndio. Mas de repente didaticamente seja melhor desse jeito.

Reply:
Não tô dizendo que precisa ser no gerúndio mesmo, especialmente se você dizer que é melhor didaticamente kkkk mas pra mim soa melhor.

Reply:
@stefanymiguel10 aqui também kkkk

Reply:
Entendi. Eles têm feelings diferente. Pra mim, o gerúndio faz parecer que você estará com o Readme em mãos enquanto trabalha com o SDK, enquanto a outra versão parece que é um call-to-action para ele usar a SDK. Você sente o mesmo?